### PR TITLE
Removed Cloud Updation in VPC mode

### PR DESCRIPTION
**Pull request by abhishekbsingh**

### CODE REVIEW

File: ako-infra/avirest/networking_handler.go [hunk 1]
```diff
@@ -87,10 +87,7 @@ func AviCloudCachePopulate(client *clients.AviClient, cloudName string, skipIPAM
 		return err
 	}
 
-	if !skipIPAM {
-		AviIPAMCachePopulate(client, strings.Split(*CloudCache.IPAMProviderRef, "#")[1])
-	}
-	return nil
+	return AviIPAMCachePopulate(client, strings.Split(*CloudCache.IPAMProviderRef, "#")[1])
```
Comment (Rishabh-Baranwal) [inline ako-infra/avirest/networking_handler.go:90]: this will result in crash if IPAM is not configured in cloud

Comment (abhishekbsingh) [reply ako-infra/avirest/networking_handler.go:90]: IPAM configuration is part of initial setup required for enabling WCP. I had added skipIPAM flag for VPC case which isn't required now. Anyway, I can check for IPAM in cloud and shutdown AKO if IPAM is not present.

Comment (Rishabh-Baranwal) [reply ako-infra/avirest/networking_handler.go:90]: Yes, handle it gracefully.

File: ako-infra/ingestion/avi_infra_controller.go [hunk 2]
```diff
@@ -75,13 +80,14 @@ func (a *AviControllerInfra) VerifyAviControllerLicense() error {
...
-		utils.AviLog.Infof("Avi Controller is running with %s license, proceeding with bootup", *response.DefaultLicenseTier)
+	for _, license := range acceptedLicensesInAvi {
+		if *response.DefaultLicenseTier == license {
+			utils.AviLog.Infof("Avi Controller is running with %s license, proceeding with bootup", *response.DefaultLicenseTier)
+			return nil
+		}
 	}
-	return nil
+
+	return fmt.Errorf("avi Controller license is not in accepted list %s. License tier is: %s", acceptedLicensesInAvi, *response.DefaultLicenseTier)
```
Comment (Rishabh-Baranwal) [inline ako-infra/ingestion/avi_infra_controller.go:90]: Case change avi -> Avi

File: ako-infra/avirest/networking_handler.go [hunk 3]
```diff
@@ -87,10 +87,10 @@ func AviCloudCachePopulate(client *clients.AviClient, cloudName string, skipIPAM
 		return err
 	}
 
-	if !skipIPAM {
-		AviIPAMCachePopulate(client, strings.Split(*CloudCache.IPAMProviderRef, "#")[1])
+	if *CloudCache.IPAMProviderRef == "" {
```
Comment (Rishabh-Baranwal) [inline ako-infra/avirest/networking_handler.go:90]: shouldn't be a nil check?

**Review by Rishabh-Baranwal — APPROVED** [review]

**Pull request merged**